app/src/main/AndroidManifest.xml

```diff
@@ -18,29 +18,35 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".DirectionsActivity"
-            android:theme="@style/AppTheme.NoActionBar"></activity>
+            android:theme="@style/AppTheme.NoActionBar"
+            android:configChanges="orientation"
+            android:screenOrientation="portrait"
+            tools:ignore="LockedOrientationActivity" />
```

Comment:
i'd prefer something more efficient than copy/pasting code in each activity but hey.. if it works it works..............